core/include/userver/engine/task/task_context_holder.hpp

```diff
@@ -18,7 +18,7 @@ namespace engine::impl {
 
 class TaskContext;
 
-using Payload = std::shared_ptr<utils::impl::WrappedCallBase>;
+using Payload = std::unique_ptr<utils::impl::WrappedCallBase>;
```

Comment:
Looks like we do not use it any more

Reply:
Used 6 times in `TaskContext` and `TaskContextHolder`